naff/client/utils/text_utils.py

```diff
@@ -0,0 +1,20 @@
+import re
+import naff.models as models
+
+__all__ = ("mentions",)
+
+
+def mentions(text: str, query: "str | re.Pattern[str] | models.BaseUser | models.BaseChannel | models.Role") -> bool:
+    """Checks whether a query is present in a text."""
+    # don't use match/case witch type(query) since subclasses aren't checked
```

Comment:
You need to add the parameters to the docstring

Reply:
I'll add it later *(I'm out for the day, should have time in the evening (UTC+2))*

Reply:
added with [728bb03](https://github.com/NAFTeam/NAFF/pull/700/commits/728bb038c0bec1c8d7f0a680965238a9089d489e)